SWLOR.Game.Server/Placeable/ControlTower/OnDamaged.cs

```diff
@@ -50,10 +54,21 @@ public bool Run(params object[] args)
             PCBaseStructure structure = _data.Single<PCBaseStructure>(x => x.ID == new Guid(structureID));
             int maxShieldHP = _base.CalculateMaxShieldHP(structure);
             PCBase pcBase = _data.Get<PCBase>(structure.PCBaseID);
+            var playerIDs = _data.Where<PCBasePermission>(x => x.PCBaseID == structure.PCBaseID).Select(s => s.PlayerID);
+            var toNotify = NWModule.Get().Players.Where(x => playerIDs.Contains(x.GlobalID));
+            DateTime timer = DateTime.UtcNow.AddSeconds(30);
+            string clock = timer.ToString(CultureInfo.InvariantCulture);
+            string sector = _base.GetSectorOfLocation(attacker.Location);
+            if (DateTime.UtcNow <= DateTime.Parse(clock))
+            {
+                foreach(NWPlayer player in toNotify)
+                {
+                    player.SendMessage("Your base in " + attacker.Area.Name + " " + sector + "Is under Attack!");
```

Comment:
Typo here. "Is under Attack!" should be " is under attack!". Aside from the capital A and I, it will run up against the sector so it would be like "Your base in Area 1 [SW]Is under Attack!"

I'll merge this in and update the text but just wanted to point it out.